fabric-item-api-v1/src/main/java/net/fabricmc/fabric/api/item/v1/elytra/FabricElytraExtensions.java

```diff
@@ -0,0 +1,37 @@
+/*
+ * Copyright (c) 2016, 2017, 2018, 2019 FabricMC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package net.fabricmc.fabric.api.item.v1.elytra;
+
+import net.minecraft.entity.LivingEntity;
+import net.minecraft.item.ElytraItem;
+import net.minecraft.item.ItemStack;
+
+/**
+ * An interface to implement on all custom elytras.
+ */
+public interface FabricElytraExtensions {
```

Comment:
maybe an better name here?

Reply:
This follows what i was told to do with the bow in #1210

Reply:
ah ok

Reply:
I like fornite guys

Reply:
Couldn't this be called `FabricElytraItem`?